### DISCUSSION TITLE
Detail list - not showing records based on table...

### DISCUSSION THREAD

**Discussion by knyklu** [microsoft/powercat-creator-kit#411] [Q&A]
...but if a create collection based on this table and set them as datasource for the list all is OK. Any Idea what is happen. Its started 2 days ago and only for list where I've change the source.

![image](https://github.com/microsoft/powercat-creator-kit/assets/134507744/d0d3354b-67f2-4265-b2ad-72a8d8d06fa2)

**Comment by yaseralhosani** [comment]
Hi @knyklu,

It can be tricky with names—make sure your column names in the Collect function match perfectly with your data source. Even small things like capitalization or a missed letter can throw things off in columns items for Detaillist  Please review carefully for columns name / data... etc. 

If the issue persists, would you be able to provide a screenshot or share the formulas for the records and Columns_Items?
